packages/cohere/src/cohere-chat-language-model.ts

```diff
@@ -218,6 +233,29 @@ export class CohereChatLanguageModel implements LanguageModelV1 {
                 return;
               }
 
+              case 'tool-calls-generation': {
+                for (const toolCall of value.tool_calls) {
+                  controller.enqueue({
+                    type: 'tool-call',
+                    toolCallId: generateId(),
+                    toolName: toolCall.name,
+                    toolCallType: 'function',
+                    args: JSON.stringify(toolCall.parameters),
+                  });
+                }
+                return;
+              }
+
+              case 'tool-calls-chunk': {
+                if (value.text) {
+                  controller.enqueue({
+                    type: 'text-delta',
+                    textDelta: value.text,
+                  });
+                }
+                return;
+              }
```

Comment:
wouldn't this need to be a `tool-call-delta`?

See https://github.com/vercel/ai/blob/main/packages/openai/src/openai-chat-language-model.ts#L461

Reply:
updated!